setup.py

```diff
@@ -263,23 +234,11 @@ def finalize_options(self):
 ]
 
 
-setup_requires = []
-
-if (not (_ROOT / 'edgedb' / 'protocol' / 'protocol.c').exists() or
-        '--cython-always' in sys.argv):
-    # No Cython output, require Cython to build.
-    setup_requires.append(CYTHON_DEPENDENCY)
-
-
-with open(str(_ROOT / 'README.rst')) as f:
-    readme = f.read()
```

Comment:
This was a feature that when installing from sdist tarballs, Cython is NOT installed (temporarily and then removed) as we deliver the `.c` files too.

Reply:
Got it. It would be nice if this project used PEP 570 build system as would suggest @fantix